services/index/src/api.js

```diff
@@ -84,6 +86,9 @@ builder.declare({
   ].join('\n'),
 }, async function(req, res) {
   let namespace = req.params.namespace || '';
+  await req.authorize({
```

Comment:
Since the only param in here is in the route we don't need the `req.authorize` anymore, right? Or is it required here due to the optional nature of `namespace`?

Reply:
Agreed. I can remove that.